File: android/app/src/main/java/com/momentum/app/models/ExerciseReel.kt
package com.momentum.app.models

import java.util.UUID

data class ExerciseReel(
    val id: String = UUID.randomUUID().toString(),
    val title: String,
    val description: String,
    val duration: String,
    val category: ReelCategory,
    val iconName: String = "figure.run",
    val colorHex: String = "#4FC3F7",
    val scienceNote: String? = null
)

enum class ReelCategory(val label: String) {
    Quick("Quick"),
    Strength("Strength"),
    Cardio("Cardio"),
    Yoga("Yoga"),
    Stretch("Stretch"),
    HIIT("HIIT"),
    Walk("Walk"),
    Mindful("Mindful")
}

File: ios/Momentum/Models/ExerciseReel.swift
import Foundation

struct ExerciseReel: Codable, Identifiable {
    let id: UUID; let title: String; let description: String; let duration: TimeInterval; let category: ReelCategory; let iconName: String; let colorHex: String; let scienceNote: String
    enum ReelCategory: String, Codable { case strength = "Strength"; case mobility = "Mobility"; case cardio = "Cardio"; case mindfulness = "Mindfulness"; case recovery = "Recovery" }
}
